common/source/docs/common-lua-scripts.rst

```diff
@@ -182,8 +194,28 @@ The ahrs library represents the Attitude Heading Reference System computed by th
 
 - :code:`groundspeed_vector()` - Returns a Vector2f containing the vehicles velocity in meters/second in north and east components.
 
-Battery
-~~~~~~~
+- :code:`get_velocity_NED()` - Returns nil, or a Vector3f containing the current NED vehicle velocity.
+
+- :code:`get_velocity_NE()` - Returns nil, or a Vector2f containing the current NE vehicle velocity.
+
+- :code:`home_is_set()` - Returns a true if home position has been set.
+
+- :code:`prearm_healthy()` - Returns a true if current pre-arm checks are passing.
+
+Arming (arming:)
+~~~~~~~~~~~~~~~~
+
+The Arming library provides access to arming status and commands.
+
+- :code:`disarm()` - Attempts to disarm the vehicle. Returns true if successful.
```

Comment:
This can return false if the vehicle was already disarmed (on some vehicles). The more useful information here is the vehicle will always be disarmed after calling this.

Reply:
is the vehicle ALWAYS disarmed? I thought you could not disarm under certain conditions...if it always disarms, will update...changed re returning false if already disarmed...

Reply:
confirmed that this method always disarms....disarming is prevented in numerous cases using rudder disarm but this method...fixed